Sources/Utility/CollectionExtensions.swift

```diff
@@ -34,12 +34,12 @@ extension Collection where Iterator.Element : Equatable {
 }
 
 extension Collection {
-    /// Splits into two arrays, first containing elements which matching the predicate
+    /// Classifies into two arrays, first containing elements which matching the predicate
     /// and other containting elements not matching the predicate.
     ///
     /// - Parameter isMatching: The predicate to apply.
     /// - Returns: Tuple of split arrays.
-    public func split(_ isMatching: (Iterator.Element) -> Bool) -> ([Iterator.Element], [Iterator.Element]) {
+    public func classify(_ isMatching: (Iterator.Element) -> Bool) -> ([Iterator.Element], [Iterator.Element]) {
```

Comment:
I believe Ruby spells this particular method as `partition`

Reply:
Sounds good to me!